lib/gitarro/backend.rb

```diff
@@ -174,6 +171,20 @@ def magicword(pr_number, context)
 
   private
 
+  # Show a message stating if there are opened PRs or not
+  def print_pr_resume(prs)
+    if pr_list_empty?(prs)
```

Comment:
here we can reverse the if and spare  1 nested stuff.
```ruby
puts '' [PRS=TRUE] Pull Requests opened. Analyzing them...  ''unless pr_list_empty?(prs)
# here then
if  @options[:changed_since] >= 0
```

Reply:
Ok, but I can't make a oneliner for the 'unless' since I need to exit the function at once with a return (otherwise) it will show one of the other messages as well.